ts/session/messages/outgoing/visibleMessage/VisibleMessage.ts

```diff
@@ -113,7 +114,7 @@ export class VisibleMessage extends ExpirableMessage {
   }
 
   public dataProto(): SignalService.DataMessage {
-    const dataMessage = super.dataProto();
+    const dataMessage = new SignalService.DataMessage({});
```

Comment:
Because `VisibleMessage` used to extend `ExpirableMessage`, but don't anymore. It extends `DataMessage` now.

![image](https://github.com/user-attachments/assets/80f47983-6aaa-46db-88d4-873e4059328c)
Which has only an abstract definition of `dataProto()`

```ts
export abstract class DataMessage extends ExpirableMessage {
  public contentProto(): SignalService.Content {
    return new SignalService.Content({
      ...super.contentProto(),
      dataMessage: this.dataProto(),
    });
  }

  public abstract dataProto(): SignalService.DataMessage;
}

```

Reply:
Cool